src/main/java/org/opensearch/timeseries/model/Job.java

```diff
@@ -31,24 +31,22 @@
 import org.opensearch.jobscheduler.spi.schedule.IntervalSchedule;
 import org.opensearch.jobscheduler.spi.schedule.Schedule;
 import org.opensearch.jobscheduler.spi.schedule.ScheduleParser;
-import org.opensearch.timeseries.model.IntervalTimeConfiguration;
-import org.opensearch.timeseries.model.TimeConfiguration;
 import org.opensearch.timeseries.util.ParseUtils;
 
 import com.google.common.base.Objects;
 
 /**
  * Anomaly detector job.
  */
-public class AnomalyDetectorJob implements Writeable, ToXContentObject, ScheduledJobParameter {
+public class Job implements Writeable, ToXContentObject, ScheduledJobParameter {
```

Comment:
Has it been decided if we should have an AnomalyDetectorJob and Forecaster Job extending this Job class or use this for both? General should probably be okay to use this for both with some checks for the types

Reply:
I think forecasting and AD share the this class. I will add a type field to job and based on type different [job processors](https://github.com/kaituo/anomaly-detection-1/blob/forecasting11/src/main/java/org/opensearch/timeseries/JobProcessor.java) are running.